recipes/sagemaker-tensorflow-container/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sagemaker-tensorflow-container" %}
+{% set version = "2.0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/aws/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: f7a92a8bb0264379ac64d1de61706575d9659234e08b5008ba6fb9f86909598f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
```

Comment:
You can make this package `noarch: python` and remove this skip on win. The package will not be installable on windows because `sagemaker_containers` doesn't have a package on windows.

Reply:
@isuruf I understand what that is doing, and why it works, but i'm confused why you would do that over just including the selector? then, if `sagemaker_containers` builds for windows, changing this recipe accordingly?

Reply:
See https://github.com/conda-forge/staged-recipes/pull/10129#discussion_r345392750